server/pulp/server/db/model/__init__.py

```diff
@@ -700,3 +700,21 @@ class LazyCatalogEntry(Document):
     checksum = StringField()
     checksum_algorithm = StringField(regex=ALG_REGEX)
     data = DictField()
+
+
+class DeferredDownload(Document):
+    """
+    A collection of units that have been handled by the streamer in the
+    passive lazy workflow that Pulp should download.
+
+    :ivar unit_id:      The associated content unit ID.
+    :type unit_id:      str
+    :ivar unit_type_id: The associated content unit type.
+    :type unit_type_id: str
+    """
+
+    # For backward compatibility
+    _ns = StringField(default='deferred_download')
```

Comment:
All concrete classes still need the `_ns` defined. It's confusing because we've talked about getting rid of it and gone back and for. For now we're going to keep it because otherwise we would have to also write a migration that affects all collections and it would be a change to the API. We have a task to remove all of these with Pulp 3.0.